pallets/ah-migrator/src/claims.rs

```diff
@@ -96,27 +96,27 @@ impl<T: Config> crate::types::AhMigrationCheck for ClaimsMigrator<T> {
 		// "Assert storage 'Claims::Total::ah_pre::empty'"
 		assert!(
 			!pallet_claims::Total::<T>::exists(),
-			"Claims total should be empty before migration starts"
+			"Assert storage 'Claims::Total::ah_pre::empty'"
 		);
 		// "Assert storage 'Claims::Claims::ah_pre::empty'"
 		assert!(
 			alias::Claims::<T>::iter().next().is_none(),
-			"Claims should be empty before migration starts"
+			"Assert storage 'Claims::Claims::ah_pre::empty'"
```

Comment:
You can remove the commend if you add it here, the script only checks if the string appears anywhere.